performance-storage-service/pss_project/api/models/rest/utils.py

```diff
@@ -5,3 +5,16 @@ def convert_environment_to_dict(environment):
         'cpu_socket': environment.cpu_socket
     }
     return db_formatted_environment
+
+
+def to_dict(obj):
```

Comment:
Is this the same as calling `.__dict__`on an object?

Reply:
Not exactly, the key difference is that this one will do the `__dict__` recursively